gradle/libs.versions.toml

```diff
@@ -45,6 +45,7 @@ findbugs-jsr305 = "3.0.2"
 flink118 =  { strictly = "1.18.1"}
 flink119 =  { strictly = "1.19.1"}
 flink120 = { strictly = "1.20.0"}
+flink20 = { strictly = "2.0-preview1"}
```

Comment:
What is the planned timeline for the normal release?

Reply:
It has just been released. The artifacts are not yet distributed yet.

Reply:
we can update the PR when the artifacts are published.

Reply:
Artifacts are published: https://flink.apache.org/downloads/